core_lib_generator/core_lib_generator_from_yaml.py

```diff
@@ -200,6 +200,10 @@ def generate_tests(self):
             {},
             TestConfigOverrideGenerateTemplate(),
         )
+        self._generate_template(f'{self.snake_core_lib_name}/tests/test_data/helpers/util.py',

```

Comment:
create a seperate generator for util, `TestUtilGenerateTemplate` this class will be responsible to generate the util and replace the template strings with the actual core lib name